shared/actions/login/index.js

```diff
@@ -44,7 +44,7 @@ export function navBasedOnLoginState () :AsyncAction {
 
     // No status?
     if (!status || !Object.keys(status).length || !extendedConfig || !Object.keys(extendedConfig).length ||
-      !extendedConfig.device) { // Not provisioned?
+      !extendedConfig.defaultDeviceID) { // Not provisioned?
```

Comment:
Excuse my JS ignorance, but does this work for strings?  

    extendedConfig.defaultDeviceID.length == 0?